skema/skema-rs/comment_extraction/src/bin/extract_comments.rs

```diff
@@ -37,15 +39,23 @@ fn main() {
         if let Some(path) = args.output {
             write(path, comments).expect("Unable to write to file!");
         } else {
-            println!("{comments}");
+            println!("{:?}", comments);
         }
     } else if extension == "py" {
         let comments = get_python_comments(input);
         let comments = serde_json::to_string(&comments).unwrap();
         if let Some(path) = args.output {
             write(path, comments).expect("Unable to write to file!");
         } else {
-            println!("{comments}");
+            println!("{:?}", comments);
```

Comment:
Same here - we want to print actual JSON to be consumed by downstream programs rather than the debug output.

Reply:
Done.